spec/factories/divisions.rb

```diff
@@ -22,6 +22,9 @@
   factory :division do
     description { Faker::Lorem.sentence }
     name { Faker::Company.name }
+    internal_name nil
+    parent nil
```

Comment:
These don't need to be explicitly defined because they'll be `nil` if you leave them out.